components/predict-form/Predict.tsx

```diff
@@ -482,20 +381,15 @@ export function Predict({
                   }
                   maxRows={15}
                   onChange={(e) => {
-                    smartUpdateResolveBy(e.currentTarget.value)
                     updateTagsPreview(e.currentTarget.value)
                     void onChangeQuestion(e)
                   }}
-                  onKeyDown={(e) => {
-                    if (onEnterSubmit(e)) return
-                    // current value doesn't include the key just pressed! So
-                    if (e.key.length === 1) {
-                      smartUpdateResolveBy(e.currentTarget.value + e.key)
-                    }
-                  }}
+                  onKeyDown={onEnterSubmit}
                   defaultValue={questionDefaults?.title}
                   onMouseDown={(e) => e.stopPropagation()}
                   {...registerQuestion}
+                  onFocus={setQuestionInFocus.bind(null, true)}
```

Comment:
off-topic: Clever syntax, is this better than `() => setQuestionInFocus(true)` for some reason? Is this type-safe? If so then seems fine (though a little confusing to the uninitiated like me)

Reply:
There's no real difference between this and `() => setQuestionInFocus(true)`. I was confused when I first saw it too so I think it's a bit of a bad habit that I use it, I'll change it to `() => setQuestionInFocus(true)`